### PR TITLE
Portfolio setup and mobile first project by the help of figma

### PR DESCRIPTION
**Pull request by VivekRaaZ**
my first figma portfolio, figma make easy software developer work.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>Raj Portfolio</title>
+</head>
+ <body>
```
Comment (RaoAkif) [inline index.html]: The ratio of your design dimensions is perfect. But this design must fit the screen of a mobile.
This is what I can see on the screen
image

Your portfolio design is supposed to be matching with the Figma design.
To achieve this:
1. Remove `border: 5px solid palevioletred;`
2. Add/Update `margin-left: 0px;`  `margin-right: 0px;` to the `.rj`

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <main class="rj">
+   <header>
+      <a href="#"class="logo">My Logo</a>
+      <nav class="nav">
+          <div class="button"><a href="#"><img src="image/Union.png" alt="button"></a></div>
+      </nav>
+  </header>
+  <section>
+    <h1>Hey There. I'm Vivek Raj glad to see you!</h1>
```
Comment (RaoAkif) [inline index.html]: The size of this headline `<h1>Hey There. I'm Vivek Raj glad to see you!</h1>` in your design is `1.5em`, which is very small as compared to the one given in Figma design i.e. `font-size: 40px;`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p2>
+
+          <h5 class="connect">LET'S CONNECT</h5>
```
Comment (RaoAkif) [inline index.html]: The design of this element is quite different from the one provided in Figma. You have used the following style for this element.
```
color: blueviolet;
 font-size: 20px;
 font-family: Arial, Helvetica, sans-serif;
```
The actual styling is:
```
font-family: 'Poppins';
font-size: 16px;
color: #7F8CFF;
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          <ul class="icon">
+              <li><a href="https://twitter.com/BossVivekRaj"><img src="image/twitter.png" alt="twitter-icon"></a></li>
+              <li><a href="https://www.linkedin.com/in/vivek-raj-a78b92256/"><img src="image/Linkedin icon.png" alt="linkedin-icon"></a></li>
+              <li><a href="https://medium.com/"><img src="image/medium.png" alt="medium-icon"></a></li>
+              <li><a href="https://github.com/VivekRaaZ"><img src="image/github.png" alt="github-icon"></a></li>
+              <li><a href="https://www.angellist.com/"><img src="image/angellist.png" alt="angellist-icon"></a></li>
+          </ul>
```
Comment (RaoAkif) [inline index.html]: The social media icons are in your design are a little bit in the centre. But in Figma design they are aligned left. Kindly see the comparison below and try to follow the Figma design:
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image| image|

File: index.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  </section>
```
Comment (RaoAkif) [inline index.html]: After resolving the above-suggested issues, kindly readjust the background image, The comparison is given below:
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| image| image|

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @VivekRaaZ, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
+* {
```
Comment (mukhammadrizooff) [inline style.css:1]: - [REQUIRED] Good job so far! Please remove `unnecessary` white space like a [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=UcPq91N1XFqxbjD6-0) 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] Please add more information about your project to your `Pull Request` description 🙏🏼 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @VivekRaaZ 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VivekRaaZ** [comment]
Raj portfolio with figma design

**Pull request merged**